src/components/dar_collection_table/ChairActions.js

```diff
@@ -26,44 +25,49 @@ const initUserData = ({dars, elections, relevantDatasets}) => {
       map(dataset => dataset.dataSetId),
       uniq
     )(relevantDatasets);
-    const relevantEmptyDars = filter(dar => {
-      if(isEmpty(dar.elections)){ return false; }
+    const relevantDarsNoElections = filter(dar => {
       const datasetId = !isEmpty(dar.data) ? dar.data.datasetIds[0] : undefined;
-      return includes(datasetId)(relevantDatasetIds);
+      return includes(datasetId, relevantDatasetIds) ? isEmpty(dar.elections) : false;
     })(dars);
-    const relevantElections = filter((election) =>
-      includes(election.datasetId)(relevantDatasetIds)
-    )(elections);
-    return {relevantEmptyDars, relevantElections, relevantDatasetIds};
+    const relevantElections = filter((election) => {
+      //NOTE: not all elections have the dataSetId attribute tied to it (not sure why)
```

Comment:
Yes, this is something we need to look more closely into for dev/staging/prod. I dug through the db changelogs and we don't seem to have tried to back populate these values for older elections. So, we do need to be flexible in the UI until we address this on the database side.